cmd/kubeadm/app/cmd/init.go

```diff
@@ -214,6 +214,10 @@ func AddInitConfigFlags(flagSet *flag.FlagSet, cfg *kubeadmapiv1beta1.InitConfig
 		&cfg.APIServerCertSANs, "apiserver-cert-extra-sans", cfg.APIServerCertSANs,
 		`Optional extra Subject Alternative Names (SANs) to use for the API Server serving certificate. Can be both IP addresses and DNS names.`,
 	)
+	flagSet.StringVar(
```

Comment:
If this is already exposed via config, we typically only expose common overrides to flags.  It's pretty rare to override the cluster name via cli vs. passing in a config.